tests/models/whisper/test_modeling_whisper.py

```diff
@@ -2487,19 +2499,13 @@ def test_default_multilingual_transcription_short_form(self):
 
         transcription = processor.batch_decode(sequences, skip_special_tokens=False)[0]
 
-        assert (
-            transcription
-            == "<|startoftranscript|><|hi|><|transcribe|><|notimestamps|> Mirchi mein ki tene vibinda prajatiya hai<|endoftext|>"
-        )
+        assert transcription == " Mirchi mein ki tene vibinda prajatiya hai"
 
         # set task to translate
         sequences = model.generate(input_features, task="translate")
         transcription = processor.batch_decode(sequences, skip_special_tokens=False)[0]
 
-        assert (
-            transcription
-            == "<|startoftranscript|><|hi|><|translate|><|notimestamps|> How much is the difference between the girls?<|endoftext|>"
-        )
+        assert transcription == " How much is the difference between the girls?"
```

Comment:
(nit) same here
```suggestion
        self.assertEqual(transcription, " How much is the difference between the girls?")
```

Reply:
applied in commit [c805b19](https://github.com/huggingface/transformers/pull/34111/commits/c805b19f2d310d44fd3d36d5505e87af4179ea41)